queue/queue.go

```diff
@@ -0,0 +1,32 @@
+package party
+
+import (
+)
+
+type Queue struct {
+	 songList []string
+}
+
+func NewQueue() *Queue {
+	q := Queue{
+		songList:make([]string, 10),

```

Comment:
you could follow something like [the package doc'd here](https://godoc.org/github.com/golang-collections/go-datastructures/queue) or implement it as a linked list. 

There are lots of good packages available online, a quick search should turn up lots of stuff. 

Also [here's a good guide to slice operations](https://github.com/golang/go/wiki/SliceTricks)